src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/service/DatabaseWriteService.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.client.MicrosoftADUser
+
+@Service
+class DatabaseWriteService {
```

Comment:
This service will allow us to develop of the database layer separately.

The names will probably need changing.